src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,80 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY_TEXT = "supply";
+    private static final String BUY_TEXT = "buy";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String line = readFromFile(fromFileName);
+        String report = createReport(line);
+        writeToFile(toFileName, report);
+    }
+
+    public String readFromFile(String fromFileName) {
+        File incomingFile = new File(fromFileName);
+        String readString;
+        StringBuilder readerBuilder = new StringBuilder();
+        try (BufferedReader reader = new BufferedReader(new FileReader(incomingFile))) {
+            String fileLine = reader.readLine();
+            while (fileLine != null) {
+                readerBuilder.append(fileLine).append(",");
```

Comment:
`,` should be a constant
Overall it looks wrong to mix up the data into a single string without any other separators, brings us more problems, when we could have kept the line separator instead and worked with the lines that we could then split by comma and assign clear operation index, amount index, etc. 
So note this for the future

Reply:
oh ok, just a lot of different opinions I heard about this like if we use it one time or not often -> no need for constants etc